src/node.h

```diff
@@ -925,12 +925,22 @@ struct ACHHandle;
 struct NODE_EXTERN DeleteACHHandle { void operator()(ACHHandle*) const; };
 typedef std::unique_ptr<ACHHandle, DeleteACHHandle> AsyncCleanupHookHandle;
 
-NODE_EXTERN AsyncCleanupHookHandle AddEnvironmentCleanupHook(
+NODE_EXTERN ACHHandle* AddEnvironmentCleanupHookRaw(
```

Comment:
Maybe add a comment that these should not be used, or use something like `Internal` or `Impl` instead of `Raw` to communicate that fact?

Reply:
Can do.